conf/abi.xml

```diff
@@ -156,6 +156,15 @@
       <field name="accel_sp" type="struct FloatVect3 *" unit="m/s^2"/>
     </message>
 
+    <message name="RELATIVE_LOCALIZATION_EKF" id="24">
```

Comment:
Indeed, this message is not used anywhere at the moment. It was created so that it can be used by other modules that wish to do something with the relative localization data.